integration/spark2/src/main/scala/org/apache/spark/sql/CarbonSession.scala

```diff
@@ -171,19 +170,24 @@ class CarbonSession(@transient val sc: SparkContext,
    */
   private def trySearchMode(qe: QueryExecution, sse: SQLStart): DataFrame = {
     val analyzed = qe.analyzed
+    val LOG: LogService = LogServiceFactory.getLogService(classOf[CarbonSession].getName)
```

Comment:
change `classOf[CarbonSession]` to `this.getClass`

Reply:
ok, done